server/src/main/java/com/cloud/usage/UsageServiceImpl.java

```diff
@@ -213,10 +212,10 @@ public Pair<List<? extends Usage>, Integer> getUsageRecords(ListUsageRecordsCmd
         Date adjustedStartDate = computeAdjustedTime(startDate, usageTZ);
         Date adjustedEndDate = computeAdjustedTime(endDate, usageTZ);
 
-        if (s_logger.isDebugEnabled()) {
-            s_logger.debug("getting usage records for account: " + accountId + ", domainId: " + domainId + ", between " + adjustedStartDate + " and " + adjustedEndDate +
-                ", using pageSize: " + cmd.getPageSizeVal() + " and startIndex: " + cmd.getStartIndex());
-        }
+        s_logger.debug(String.format("Getting usage records for account ID [%s], domain ID [%s] between [%s] and [%s] using page size [%s] and start index [%s].",
```

Comment:
please don't remove those guards yet. we can remove those when parameter evaluation is no longer happening.